src/dbt/kipptaf/models/extracts/tableau/rpt_tableau__staff_renewal_feed.sql

```diff
@@ -0,0 +1,98 @@
+with
+    step_scales as (
+        select
+            sr.employee_number,
+            pss.salary_rule,
+            pss.scale_cy_salary,
+            pss.scale_ny_salary,
+            pss.scale_step,
+            pss.school_level,
+        from {{ ref("base_people__staff_roster") }} as sr
+        left join
+            {{ source("people", "src_people__salary_scale") }} as pss

```

Comment:
use staging model, or create one if it doesn't exist

Reply:
done